tests/integration/targets/homebrew/tasks/formulae.yml

```diff
@@ -40,15 +40,13 @@
     homebrew:
       name: "{{ package_name }}"
       state: absent
-      update_homebrew: false
```

Comment:
Why remove this from the tests? Not sure I see how it relates to the rest of the PR.

Reply:
This is the default value so I removed it in the first commit to reduce duplication a bit.